File: php/utility/ResultBody.php
<?php
declare(strict_types=1);

// BranchDailyExports SDK utility: result_body

class BranchDailyExportsResultBody
{
    public static function call(BranchDailyExportsContext $ctx): ?BranchDailyExportsResult
    {
        $response = $ctx->response;
        $result = $ctx->result;
        if ($result && $response && $response->json_func && $response->body) {
            $result->body = ($response->json_func)();
        }
        return $result;
    }
}

File: go/utility/result_body.go
package utility

import "github.com/voxgig-sdk/branch-daily-exports-sdk/go/core"

func resultBodyUtil(ctx *core.Context) *core.Result {
	response := ctx.Response
	result := ctx.Result

	if result != nil {
		if response != nil && response.JsonFunc != nil && response.Body != nil {
			json := response.JsonFunc()
			result.Body = json
		}
	}

	return result
}
